src/class-elements/private-static-method-function-call-access.case

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: PrivateName IdentifierName PropName StringValue
+info: |
+  Static Semantics: PrivateStringValue
+    PrivateName::
+      #IdentifierName
+  Return the String value consisting of the sequence of code units corresponding to PrivateName
+
+template: productions
+features: [class-methods-private]
+---*/
+
+//- fields
+#foo() { return 'test'; }
+static #bar() { return '262'; }
+//- privateinspectionfunctions
+method1() {
+  return this.#foo.call(C);
```

Comment:
This needs adding the matching reference from the spec that allows this in the desc/info above. Otherwise it's hard to tell the goal of this test. The top comment on this PR clarified it a bit for me.

This needs adding a group of tests that assert other behaviors:

- [ ] when the function is called with other values, like `this.#foo.call(this)`, `this.#foo.call({})`, `this.#foo.call(null)`, `C.#bar.call(C)`, including other values as the instance variable name (instead of `this`), `undefined`, etc etc
- [ ] other possible private methods.
- [ ] fields with function values, perhaps?